### ISSUE TITLE
Crash on FilePropertiesDialog display for FM

### ISSUE THREAD

**Issue by shuvashish76** [MuntashirAkon/AppManager#1386]
### Please check before submitting an issue

- [X] I know what my device, OS and App Manager versions are
- [X] I know how to take logs
- [X] I know how to reproduce the issue which may not be specific to my device

### Describe the bug

AM crashed while opening Owner (UID) dialog options for FileManager.

### To Reproduce

1. 3-dot menu > Labs > Files
2. Click on 3-dot menu of any folder > Properties
3. Click on pencil icon of Owner (UID)
App crashed.
One time crash only, perhaps due to trying to open it very quickly...can't reproduce it.

### Logs

```
FATAL EXCEPTION: main
Process: io.github.muntashirakon.AppManager.debug, PID: 18229
java.util.ConcurrentModificationException
	at java.util.ArrayList$Itr.next(ArrayList.java:860)
	at io.github.muntashirakon.AppManager.fm.dialogs.FilePropertiesDialogFragment.displayUidUpdater(FilePropertiesDialogFragment.java:340)
	at io.github.muntashirakon.AppManager.fm.dialogs.FilePropertiesDialogFragment.$r8$lambda$9IXBBN_mnHjB2pAJnEoqd2r4WZY(Unknown Source:0)
	at io.github.muntashirakon.AppManager.fm.dialogs.FilePropertiesDialogFragment$$ExternalSyntheticLambda4.onChanged(D8$$SyntheticClass:0)
	at androidx.lifecycle.LiveData.considerNotify(LiveData.java:133)
	at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:151)
	at androidx.lifecycle.LiveData.setValue(LiveData.java:309)
	at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
	at androidx.lifecycle.LiveData$1.run(LiveData.java:93)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

### Device info

App version: 4.0.0-beta01-DEBUG
App version code: 436
Android build version: eng.chef.20200713.073313
Android release version: 9
Android SDK version: 28
Android build ID: lineage_TB8704-userdebug 9 PQ3A.190801.002 eng.chef.20200713.073313 test-keys
Device brand: Lenovo
Device manufacturer: Lenovo
Device name: TB-8704X
Device model: Lenovo TB-8704X
Device product name: lineage_TB8704
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-IN
In-App Language: auto
Mode: root
Inferred Mode: Root

### Additional context

After the crash I clicked on AM crash report from the notification & choose to open with AM Code editor...but the code editor showed empty results. Fortunately LogFox already captured the crash as it's always running in background. Here the provided logs are from LogFox.
Option to open the crash notification on AM Code editor would be nice. Thanks.

**Comment by MuntashirAkon** [comment]
> Option to open the crash notification on AM Code editor would be nice.

I've got a different plan for handling crashes as well as acquiring logs from AM.

**Comment by MuntashirAkon** [comment]
Fixed in 1b6a7734302692e111b7ea76a792d2a8717d7c57

**Comment by shuvashish76** [comment]
> I've got a different plan for handling crashes as well as acquiring logs from AM.

Sometimes it doesn't work when choose to open with FM e.g. Material Files.
